### DISCUSSION TITLE
Импорт CSV данных в таблицы циклов

### DISCUSSION THREAD

**Discussion by dmbrtlt** [totumonline/totum-ru-issues-and-discussions#605] [Как это сделать?]
Есть таблица контрагентов (таблица циклов) и создана расчетная таблица контрагента. Вопрос как импортировать массив данных? При попытке импорта данные вносятся в таблицу циклов, но при переходе в расчетную поля пустые

**Comment by erpic-ru** [comment]
В простой таблице, куда будут загружаться данные, писать обработчик, который кодом действия будет создавать циклы и создавать записи внутри циклов

**Reply by dmbrtlt** [reply]
а можно пример такого обработчика?)

**Reply by erpic-ru** [reply]
Выбор сильно зависит от данных
 
1. Грузите данные в простую таблицу с простыми строками. 

2. Следующим шагом создаете  справочники через listUniq и делаете уже селекты. Это можно сделать кодами и пересчетом. 

4. Делаете кнопку и через Листреплейс перебираете строки. например, первым действием создаете цикл, вторым проставляете значения. Если сильно огромный массив, то можно его кроном пройти с лимитом на число записей, код где-то тут был
